apps/hash-ai-worker-py/pyproject.toml

```diff
@@ -37,6 +37,7 @@ ignore = [
     "D401", # Relates to PEP-257 but also conflicts with Google Python Style Guide, generally gets in the way
     "ANN101", # most type checkers infer the type of `self` automatically
     "ANN102", # most type checkers infer the type of `cls` automatically
+    "FA102", # Will become the default in a later Python release
```

Comment:
This is incorrect, we should be able to simply set the `target-version` option, as everything in `FA102` we're compatible with.

Reply:
about https://peps.python.org/pep-0563/, it has been replaced by https://peps.python.org/pep-0649/

Reply:
(https://docs.python.org/3/whatsnew/3.11.html#pep-563-may-not-be-the-future)

Reply:
Could you make the required steps here, please?

Reply:
Thanks!